components/net/hsts.rs

```diff
@@ -139,15 +142,160 @@ impl HstsList {
     }
 
     /// Step 10 of https://fetch.spec.whatwg.org/#concept-main-fetch.
```

Comment:
Nit: is this the right spec link?

Reply:
I did not touch this line, but it should be corrected to "Step 2.9".
> Set request’s current URL’s scheme to "https" if all of the following conditions are true:
> * request’s current URL’s scheme is "http"
> * request’s current URL’s host is a domain
>  * Matching request’s current URL’s host per Known HSTS Host Domain Name Matching results in either a superdomain match with an asserted includeSubDomains directive or a congruent match (with or without an asserted includeSubDomains directive). [HSTS]